gatsby-node.js

```diff
@@ -235,9 +235,9 @@ exports.createPages = async ({ graphql, actions: { createPage } }) => {
 
   // Generate puml to svg
   console.info(`generating svg diagrams in ${diagramOutputDir}...`);
-  execSync(
-    `set -e; cp content/plantuml/* ${diagramOutputDir}/; for f in $(find content -name '*.puml'|grep -v /plantuml/); do cp $f ${diagramOutputDir}/$(basename $(dirname $f))-$(basename $f); done; java -jar $JAVA_HOME/lib/plantuml.jar -tsvg ${diagramOutputDir}/*.puml; rm -f ${diagramOutputDir}/*.puml`
-  );
+  // execSync(
+  //   `set -e; cp content/plantuml/* ${diagramOutputDir}/; for f in $(find content -name '*.puml'|grep -v /plantuml/); do cp $f ${diagramOutputDir}/$(basename $(dirname $f))-$(basename $f); done; java -jar $JAVA_HOME/lib/plantuml.jar -tsvg ${diagramOutputDir}/*.puml; rm -f ${diagramOutputDir}/*.puml`
+  // );
```

Comment:
Everything is fine, except theses commented lines. Can you uncomment it, will be ready to merge after that :)

Reply:
fixed, thank you!